how-to/troubleshoot/autograph_credentials.rst

```diff
@@ -0,0 +1,32 @@
+Autograph Credentials
+=====================
+
+Autograph uses `hawk`_ for authentication. Each ``hawk`` user, has access to
+one or more autograph *signers*. When making a signing request,
+`signingscript`_ can choose which signer to use by specifying a ``keyid``. If
+no ``keyid`` is specified, Autograph will use the "default" signer.
+
+.. note::
+
+   It's not entirely clear how Autograph chooses a default signer. Therefore
```

Comment:
Coincidentally, @jbuck told me today that:
> if there’s no signer id configured, autograph uses the first signer id in the permission list

But unless that's visible to us, maybe it doesn't warrant a change here?

Reply:
I *think* it's visible to us via the datasette dumps (based on rowId)? But it's nebulous enough that "not entirely clear" is still accurate imo :p